types/supports-color/index.d.ts

```diff
@@ -1,9 +1,10 @@
-// Type definitions for supports-color 7.2
+// Type definitions for supports-color 8.1
 // Project: https://github.com/chalk/supports-color
 // Definitions by: Melvin Groenhoff <https://github.com/mgroenhoff>
 //                 Matt Traynham <https://github.com/mtraynham>
 //                 Piotr Błażejewicz <https://github.com/peterblazejewicz>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
+/// <reference types="node" />
```

Comment:
Personally, I'd leave a line break so it doesn't blend into the header.

Reply:
amended, thx!